.circleci/config.yml

```diff
@@ -258,6 +262,16 @@ jobs:
             if << parameters.run_in_parallel >>; then
               EXTRA_PARAMS="-n4 $EXTRA_PARAMS"
             fi;
+            if << parameters.save_perf_report >>; then
+              if [[ $CIRCLE_BRANCH == "main" ]]; then
+                git clone https://$VIP_VAP_ACCESS_TOKEN@github.com/zenithdb/zenith-perf-data.git
+                mkdir -p zenith-perf-data/data/local
+                EXTRA_PARAMS="--out-dir zenith-perf-data/data/local $EXTRA_PARAMS"
+              fi;
```

Comment:
It'd be nice to have a way to see html reports for PRs (different branches) too.

Reply:
There are a few ways to achieve that, it might be per branch report rendering or comments from bot under submitted PR with a performance summary (or both). Lets start with what is currently there and add new features on top separately